config/faf.conf.in

```diff
@@ -5,6 +5,7 @@
 PluginsDir = @sysconfdir@/faf/plugins/
 TemplatesDir = @sysconfdir@/faf/templates/
 AutoEnablePlugins = True
+Version = @fafversion@
```

Comment:
I do not think that storing the version in faf.conf is a good idea. This file is often modified by user and the file provided by package end up as faf.conf.rpmnew.

It would be better to put the version to file which will be located somewhere in /usr/share/...

Reply:
@xsuchy I tend to agree, this just seemed like the easiest solution. However, ideally, I'm not sure if we even want to store the version in a file. Since we're doing all kinds of transformations of `.in` files during build, we could just as well put the version number anywhere it's needed in the same vein. Does that sound like a better solution?